devApps/XFormsApp/MainPage.cs

```diff
@@ -28,13 +28,15 @@
 using Microsoft.IdentityModel.Clients.ActiveDirectory;
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 using System.Linq;
 using System.Reflection.Emit;
 using System.Text;
 using Xamarin.Forms;
 
 namespace XFormsApp
 {
+    [SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncVoid", Justification = "Reviewed.")]
```

Comment:
>[SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncVoid", Justification = "Reviewed.")] [](start = 4, length = 97)

pls. move to globalsuppression file if needed. 
Justification should be more meaningful, e.g. method handler signature requires void

Reply:
pls. move to globalsuppressions

---
In reply to: [168047071](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/992#discussion_r168047071) [](ancestors = 168047071)